CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## 2016-05-08 Release XX.XX.XX
+
+### Added
+
+- allow absent for ensure attribute
+
+### Fixed
+
+- Link to badges
+- Correct handle of ensure attribute for nodejs
+
+### Maintenance
+
+- partial reformat of code for better readability
```

Comment:
I would bump the module to 2.0.0 because this is a breaking change. I added a note about ruby to the `Maintenance` part.